### ISSUE TITLE
Options to align home screen icons

### ISSUE THREAD

**Issue by navdpsingh** [8bitPit/Niagara-Issues#2643]
### Is your feature request related to a problem?

I like to emphasize the home screen wallpaper but the icons, even though the are left aligned, block the visual center.

### Describe the solution you'd like

I would like the ability to control the vertical alignment of the icons to being at the top under the widgets, middle, or bottom.

### Feature Request Procedure

- [X] I have read the [knowledge base article on feature requests](https://help.niagaralauncher.app/article/44-feature-requests) and followed the steps mentioned

**Comment by NiagaraSupport** [comment]
Hey there, thanks for the request.
You can move your favorites up and down by moving the widget. Moving them individually is currently not planned, but perhaps you can find some workaround solutions. The user in #2633 uses blank icons, for example. Perhaps those are more to your liking?

**Comment by navdpsingh** [comment]
Hi, thanks for the response. I tried this solution however I dont think it'll work for me. The outcome I want is to have the time widget left top aligned and favorites left bottom aligned. I hide app names on the homescreen. I tried OLauncher and saw this formatting was available. I understand it's not in the current feature backlog, requesting it to be considered. Regards.
  

  

  
  
--
  
Thanks,
  
-Nav
  

  

        
  
  
>
